### PR TITLE
TODO: Added Examples in ask.py

### PR DESCRIPTION
**Pull request by czgdp1807**
I have added examples in the docstrings of the
following methods:
i) transcendental
ii) infinite
iii) commutative
It was given in the TODOs in the source code.

### CODE REVIEW

**Comment by Abdullahjavednesar** [comment]
Ping @debugger22

File: sympy/assumptions/ask.py [hunk 1]
```diff
@@ -241,8 +241,19 @@ def transcendental(self):
         transcendental numbers. A transcendental number is a real
         or complex number that is not algebraic.
 
+        Examples
+        ========
+
+        >>> from sympy import ask, Q, pi, exp
+        >>> ask(Q.transcendental(exp(1)))
+        >>>
```
Comment (oscargus) [inline sympy/assumptions/ask.py]: I think these empty lines messed up the tests.

File: sympy/assumptions/ask.py [hunk 2]
```diff
@@ -241,8 +241,19 @@ def transcendental(self):
...
+        True
+        >>> ask(Q.transcendental(pi)))
```
Comment (oscargus) [inline sympy/assumptions/ask.py]: Typo. Extra right bracket.

File: sympy/assumptions/ask.py [hunk 3]
```diff
@@ -241,8 +241,19 @@ def transcendental(self):
...
+        True
+        >>> ask(Q.transcendental(2)))
```
Comment (oscargus) [inline sympy/assumptions/ask.py]: Same here.

**Comment by oscargus** [comment]
If you can fix the minor issues this would be a welcome addition to the documentation.

**Comment by czgdp1807** [comment]
@oscargus I have made the https://github.com/sympy/sympy/pull/15902 addressing the same changes and I am closing this now. Thanks for reviewing this. I have fixed the typos. :-)

**Pull request closed without merging**